README.adoc

```diff
@@ -1 +1,55 @@
-See test cases
\ No newline at end of file
+# DPF_03
+
+Holds a customized lexical tokenizer as a Spark UDF and a bloom filter aggregator.
+Used to tokenize spark string columns and run Spark aggregation into a bloom filter with configurable filter size selection.
+
+## Features
+
+### TokenizerUDF
+
+Spark UDF that will tokenize incoming string value and return it as a list of byte arrays.
+Tokenization rules are set in blf_01.
+
+### ByteArrayListAsStringListUDF
+
+Spark UDF that converts results from TokenizerUDF into a list of strings.
+
+### BloomFilterAggregator
+
+Custom spark aggregator that aggregates a column string tokens into a single bloom filter and
+returns the bytes of the resulting filter that can be processed.
+
+Filter size is selected by giving the aggregator the name of a spark column that holds an estimated value of tokens and
+by configuring a map of bloom filters with preset values (expected number of items, false positive probability).
+
+
+## Documentation
+
+See the official documentation on https://docs.teragrep.com[docs.teragrep.com].
+
+## Limitations
+
+Compatible with Java version 1.8 other versions might not work
```

Comment:
"Compatible with Java version 1.8, other versions might not work." please add comma and period

Reply:
done!